packages/wdio-browserstack-service/src/accessibility-handler.ts

```diff
@@ -93,23 +99,35 @@ class _AccessibilityHandler {
         this._sessionId = sessionId
         this._accessibility = isTrue(this._getCapabilityValue(this._caps, 'accessibility', 'browserstack.accessibility'))
 
-        if (isBrowserstackSession(this._browser) && isAccessibilityAutomationSession(this._accessibility)) {
+        if (isBrowserstackSession(this._browser) && isAccessibilityAutomationSession(this._accessibility) && !this.isAppAutomate) {
             const deviceName = this._getCapabilityValue(this._caps, 'deviceName', 'device')
             const chromeOptions = this._getCapabilityValue(this._caps, 'goog:chromeOptions', '')
 
             this._accessibility = validateCapsWithA11y(deviceName, this._platformA11yMeta, chromeOptions)
         }
 
+        if (isBrowserstackSession(this._browser) && isAppAccessibilityAutomationSession(this._accessibility, this.isAppAutomate)) {
+            const deviceName = this._getCapabilityValue(this._caps, 'deviceName', 'device')
```

Comment:
we can remove this, we don't have any validation on deviceName

Reply:
Done